src/processing_eeg_methods/new_share.py

```diff
@@ -0,0 +1,390 @@
+from dataclasses import dataclass
+from pathlib import Path
+from typing import List
+
+@dataclass
+class DatasetInfo:
```

Comment:
The `new_share` looks excellent; however, preferably do not mix PRs. This one is only for the `Standard Saving Path`, and it is ready to merge. The new_share is not ready yet and needs to be handled separately.